Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -47,10 +48,7 @@ and composition with linear maps `f`, `Q.comp f x = Q (f x)`.
 
 ## Notation
 
-In this file, the variable `R` is used when a `Ring` structure is sufficient and
-`R₁` is used when specifically a `CommRing` is required. This allows us to keep
-`[Module R M]` and `[Module R₁ M]` assumptions in the variables without
-confusion between `*` from `Ring` and `*` from `CommRing`.
+In this file, the variable `R` is used when a `CommSemiring` structure.
 
 The variable `S` is used when `R` itself has a `•` action.
```

Comment:
Let's point out that we cannot just generalize for free:
```suggestion
The variable `S` is used when `R` itself has a `•` action.

## Implementation notes

While the definition and many results make sense if we drop commutativity assumptions,
the correct definition of a quadratic form in the noncommutative setting would require
substantial refactors from the current version.
Zulip thread: https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/Quadratic.20Maps

```

Reply:
I slightly tweaked your suggestion, can you take one last look?